openapi/openapi.yaml

```diff
@@ -931,6 +930,243 @@ paths:
+          description: >
+            Type of environment (runtime or buildtime) which is being analyzed.
+          schema:
+            type: string
+            enum:
+              - buildtime
+              - runtime
+            default: runtime
+        - name: origin
+          in: query
+          required: false
+          description: >
+            A remote where the image is being used. This is used for tracking as well as
+            for automated reporting when results are available.
+          schema:
+            type: string
+        - name: debug
+          in: query
+          required: false
+          description: >
+            Run the given analyzer in a verbose mode so developers can debug
+            analyzer.
+          schema:
+            type: boolean
+            default: false
+        - name: verify_tls
+          in: query
+          required: false
+          description: >
+            Verify TLS certificates of registry from where images are
+            pulled from.
+          schema:
+            type: boolean
+            default: true
+        - name: force
+          in: query
+          required: false
+          schema:
+            type: boolean
+            default: false
+          description: >
+            Do not use cached results, always run analysis.
+      requestBody:
+        required: true
+        description: Build log to be stored.
+        content:
+          application/json:
+            schema:
+              x-body-name: log_info
+              $ref: "#/components/schemas/Log"
+      responses:
+        "202":
+          description: Successful response with analyzer id.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AnalysisResponse"
+        "400":
+          description: On invalid request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AnalysisResponseError"
+
+  /buildlog-analyze:
+    post:
+      tags: [Buildlogs Analysis]
+      x-openapi-router-controller: thoth.user_api.api_v1
+      operationId: post_buildlog_analyze
+      summary: Analyze the given build log.
+      parameters:
+        - name: output_format
+          in: query
+          required: false
+          description: Type of output format which build analyser returns.
+          schema:
+            type: string
+            enum:
+              - json
+              - dict
+              - html
+              - plain
+            default: json
+            format: string
+        - name: handler
+          in: query
+          required: false
+          description: Handler to parse log dependencies.
+          schema:
+            type: string
+            enum:
+              - pip
+              - pipenv
+            format: string
+        - name: limit
+          in: query
+          required: false
+          description: Limit number of candidates.
```

Comment:
Can this be automatically determined/configured to a sane default? This is nothing a user of Thoth would provide/can derive.